coverage_config_x86_64.json

```diff
@@ -1,5 +1,5 @@
 {
-  "coverage_score": 75.0,
+  "coverage_score": 73.30,
```

Comment:
Can we try to call the new API in `vhost-user-backend/src/handler.rs` or it is too complicated?
If it is the case, can you explain it in the commit description?

Reply:
The whole `handler.rs` is not tested and I assume this is because it is too difficult to create the handler for the test. I will update the commit message to mention this.

Reply:
Sorry, I posted the wrong path, I meant if we can add the test here: `vhost-user-backend/tests/vhost-user-server.rs` , but since the backend is the dummy one, I'm not sure how much we can increase the coverage.

Reply:
The `MockVhostBackend` does not use `VhostUserHandler`, so it can not be tested without rewriting the `MockVhostBackend` and all of the tests.

Reply:
Okay, thanks for checking!